test/plugins/validation/2and3/responses.js

```diff
@@ -208,6 +245,53 @@ describe('validation plugin - semantic - responses', function() {
         expect(res.errors.length).toEqual(0);
       });
 
+      it('should complain about an inline schema when using oneOf, anyOf, or allOf', function() {
+        const spec = {
+          paths: {
+            '/stuff': {
+              get: {
+                summary: 'list stuff',
+                operationId: 'listStuff',
+                responses: {
+                  200: {
+                    description: 'successful operation',
+                    content: {
+                      'application/json': {
+                        schema: {
+                          oneOf: [
+                            {
+                              type: 'object'
+                            }
+                          ]
```

Comment:
I think it would be good to add one more element to the `oneOf` list that is a `$ref`, so you can see that it only flags the problem schemas.